submissions/lawalNurudeen/html-forms/contact-form.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Contact-Form</title>
+</head>
+<body>
+    <form action="#" method="POST" target="_self" enctype="multipart/form-data">
```

Comment:
@Nurmalee Remove the elements `action`, `method` and `enctype` they are not needed

Reply:
Requested alterations have been applied..